### PR TITLE
Revert Unsupported Testnets To Mainnet

### PR DESCRIPTION
**Pull request by derHowie**
## What changed (plus any additional context for devs)
This PR is in support of #4001 : we want to handle situations where a user has a deprecated testnet set in state. This code gracefully reverts back to mainnet. I separated this onto its own branch for ease of testing.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/screens/WalletScreen.js
Comment (estebanmino) [inline src/screens/WalletScreen.js:107]: why not just calling `revertToMainnet` here?

**Review by estebanmino — APPROVED** [review]
changes lgtm, looking good on the app as well, should we wait for to be ready to go and merge it right before merging https://github.com/rainbow-me/rainbow/pull/4001/files# ?

**Review by estebanmino — APPROVED** [review]
looking good! https://www.loom.com/share/340739393b2a43ca85239cda26f711c1

File: src/screens/WalletScreen.js
Comment (skylarbarrera) [reply src/screens/WalletScreen.js:107]: +1 hurr

**Review by skylarbarrera — APPROVED** [review]
working good

**Review by brunobar79 — APPROVED** [review]

**Pull request merged**
